### PR TITLE
Update on Diff tree UX

### PR DESCRIPTION
**Pull request by bilalabbad**
- On diff tree, we now have model name in tree root level
- New colors on diff 

image

### CODE REVIEW

File: frontend/app/src/screens/diff/diff-tree.tsx [hunk 1]
```diff
@@ -16,11 +18,19 @@ export default function DiffTree({ nodes, loading, ...props }: DiffTreeProps) {
   const navigate = useNavigate();
 
   useEffect(() => {
+    let rootModelMap: Record<string, Array<string>> = {};
+
     const formattedNodes: TreeProps["data"] = nodes.flatMap((node: DiffNode) => {
+      const currentModel = rootModelMap[node.kind];
+      rootModelMap = {
```
Comment (pa-lem) [inline frontend/app/src/screens/diff/diff-tree.tsx]: is it possible to avoid variable re definition for a more functional way?

**Review by pa-lem — APPROVED** [review]

**Comment by pa-lem** [comment]
with a screenshots it will help to documente the PR and know what's visually updated

File: frontend/app/src/screens/diff/diff-tree.tsx (hunk 1 above)
Comment (bilalabbad) [reply frontend/app/src/screens/diff/diff-tree.tsx]: tbh, I feel like mutation here is ok and explicit enough. I did some renaming update to make code a bit clearer

**Comment by dgarros** [comment]
This PR need to go into the release-0.16 branch

**Pull request merged**